tests/PerformanceHarness/performance_test_basic.py

```diff
@@ -258,6 +259,22 @@ def removeAllArtifactsExceptFinalReport():
         except OSError as error:
             print(error)
 
+    def testDirsCleanupState(self):
+        try:
+            def removeArtifacts(path):
+                print(f"Checking if test artifacts dir exists: {path}")
+                if Path(path).is_dir():
+                    print(f"Cleaning up test artifacts dir and all contents of: {path}")
+                    shutil.rmtree(f"{path}")
+
+            stateDirs = list(Path(self.varLogsDirPath).rglob("state"))
+            print(f"stateDirs: {stateDirs}")
+            for stateDirPath in stateDirs:
```

Comment:
Would it be useful to chack that the directory contains a `shared_memory.bin` file, or is the fact that the directory is named `state` good enough?

Reply:
I was just wondering if there could be a directory named `state` used for some other purpose, and that we would not want to delete?

Reply:
Good point.  I've narrowed the search to the `node_*` log directories. So this should be narrow enough now since the only `state` directories within are what we're looking to remove else it would be a collision.

Addressed here: [6348e9a](https://github.com/AntelopeIO/leap/pull/1679/commits/6348e9ac253be1bb1b082293987242bd81280725)